src/app/analyze/stackreports/container-toggler/container-toggler.component.ts

```diff
@@ -0,0 +1,25 @@
+import {Component, ViewEncapsulation, Output, EventEmitter} from '@angular/core';
```

Comment:
I removed the stackreports directory, please merge these changes in to the stack-details module